src/nvim/regexp_nfa.c

```diff
@@ -6075,6 +6079,13 @@ static int nfa_regmatch(nfa_regprog_T *prog, nfa_state_T *start, regsubs_T *subm
     if (got_int) {
```

Comment:
The original patch changes the line above this (Github won't let me comment on it) from `fast_breakcheck` to `line_breakcheck` (i.e., wait less time before calling `os_breakcheck`).  However, if I make that change, the unit tests segfault in the event queue code.